docs/exploration-and-organization/exploration.md

```diff
@@ -10,6 +10,40 @@ redirect_from:
 
 As long as you're not the very first user in your team's Metabase, the easiest way to start exploring your data is by looking at dashboards, charts, and lists that your teammates have already created. The best place to start is by checking out any dashboards that might be pinned on your home page, or in [collections][collections] you have access to.
 
+## Command palette
+
+The command palette lets you jump to anywhere in your Metabase.
+
+To bring up the command palette, hit cmd/ctrl + k. That's `cmd + k` for Macs, or `ctrl + k` for Windows PCs.
+
+You can use the command palette to:
+
+- Create new things (like new questions and dashboards)
+- Find things (saved questions, dashboards, models, etc.)
+- Find and jump to Admin settings
+
+So anytime you want to do or find anything in Metabase, just hit cmd/ctrl + k and start typing what you want to do.
+
+## Search
+
+If you want to refine your search with filters, use the search bar and click through to the search results page.
```

Comment:
This will change on v50.

We won't have the search bar anymore. We will have a single search button that opens the command palette.
https://github.com/metabase/metabase/issues/42346